tool/lib/utils.dart

```diff
@@ -109,26 +109,24 @@ class CliCommand {
   final bool throwOnException;
 }
 
+typedef DevToolsProcessResult = ({int exitCode, String stdout, String stderr});
+
 extension DevToolsProcessManagerExtension on ProcessManager {
-  Future<String> runProcess(
+  Future<DevToolsProcessResult> runProcess(
     CliCommand command, {
     String? workingDirectory,
     String? additionalErrorMessage = '',
   }) async {
-    String stdout = '';
+    final processStdout = StringBuffer();
+    final processStderr = StringBuffer();
 
     final process = await spawn(
       command.exe,
       command.args,
       workingDirectory: workingDirectory,
     );
-    unawaited(
-      process.stdout
-          .transform(
-            utf8.decoder,
-          )
-          .forEach((x) => stdout = '$stdout$x'),
-    );
+    process.stdout.transform(utf8.decoder).listen(processStdout.write);
+    process.stderr.transform(utf8.decoder).listen(processStderr.write);
```

Comment:
maybe I'm mistaken but I don't think we need these. `ProcessManager.spawn` returns `_ForwardingSpawn` which should handle stdout and stderr